pubsublite/streaming-analytics/README.md

```diff
@@ -0,0 +1,107 @@
+# Pub/Sub Lite with Cloud Dataflow
+
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=pubsublite/streaming-analytics/README.md)
+
+Samples showing how to use [Pub/Sub Lite] with [Cloud Dataflow].
+
+## Pub/Sub Lite to Cloud Storage sample
+
+[PubsubliteToGcs.java](examples/PubsubliteToGcs.java)
+
+This sample shows how to create an [Apache Beam] streaming pipeline that reads
+messages from [Pub/Sub Lite], group the messages using a fixed-sized windowing
+function, and writes them to [Cloud Storage].
+
+Resources needed for this example:
+
+1. A pair of Pub/Sub Lite topic and subscription. 
+2. A Cloud Storage bucket.
+
+### Setting up
+
+1. [Enable the APIs](https://console.cloud.google.com/flows/enableapi?apiid=dataflow,compute_component,logging,storage_api,pubsublite.googleapis.com): Cloud Dataflow, Compute Engine, Cloud Logging, Cloud Storage, Pub/Sub Lite.
+
+1. Create a Cloud Storage bucket.
+
+   ```bash
+   PROJECT=$(gcloud config get-value project)
+   BUCKET=your-gcs-bucket
+   
+   gsutil mb gs://$BUCKET
+   ```
+   
+ 1. Create a Pub/Sub Lite topic and subscription. Set `LITE_LOCATION` to a [Pub/Sub Lite location].
+ 
+    ```bash
+    TOPIC=your-lite-topic
+    SUBSCRIPTION=your-lite-subscription
+    LITE_LOCATION=your-lite-location
+    
+    gcloud pubsub lite-topics create $TOPIC --zone=$LITE_LOCATION --partitions=1 \
```

Comment:
nit: [style] completely optional, if the parameters span more than one line I some times find it easier to read if they're each in their own line.

```
gcloud pubsub lite-topics create $TOPIC \
  --zone=$LITE_LOCATION \
  --partitions=1 \
  --per-partition-bytes=30GiB
```

However, if `--partitions` and `--per-partition-bytes` are optional and have default values in `gcloud`, I think I would prefer dropping them for the sample and just adding a note with a link where they can look how to set them up if they need different settings that the defaults.